module/metrics/labels.go

```diff
@@ -113,6 +113,7 @@ const (
 	ResourceFollowerLoopCertifiedBlocksChannel    = "follower_loop_certified_blocks_channel"    // follower loop, certified blocks buffered channel
 	ResourceClusterBlockProposalQueue             = "cluster_compliance_proposal_queue"         // collection node, compliance engine
 	ResourceTransactionIngestQueue                = "ingest_transaction_queue"                  // collection node, ingest engine
+	ResourceSubmitCollectionGuaranteesQueue       = "submit_col_guarantee_queue"                // collection node, pusher engine
```

Comment:
```suggestion
	ResourceSubmitCollectionGuaranteesQueue       = "pusher_col_guarantee_queue"                // collection node, pusher engine
```
I anticipate we will be removing the `SubmitCollectionGuarantee` message type as mentioned. Including the engine name here instead may be more informative.